SU-HSDO/suhumsci
# SHS-5052: Video preview in CKeditor should better reflect rendered size

# READY FOR REVIEW

## Summary
When embed a video in a CKEditor textarea and choosing the "Align center and break text" alignment, then the video preview is the full width of the CKEditor area to indicate that it will be rendered at full width.

## Need Review By (Date)
02/09

## Urgency
medium

## Steps to Test on Colorful and Traditional
1. Go to any page, edit it or create a new one
2. In a WYSIWYG area, embed a video and select the "Align center and break text" alignment. 
3. The preview of the video should be the full width of the CKEditor area

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
